src/shared/Button.tsx

```diff
@@ -1,13 +1,10 @@
 import React, { FC, ReactElement } from 'react';
-import { BaseButton } from '../styles/components';
 
 type ButtonProps = {lable: string, disabled?: boolean, onClick: () => void};
 
-const Button: FC<ButtonProps> = ({lable, disabled = false, onClick}): ReactElement => {
+export const Button: FC<ButtonProps> = ({lable, disabled = false, onClick}): ReactElement => {
 
     return (
-      <BaseButton disabled={disabled} onClick={onClick}>{lable}</BaseButton>
+      <button className='primery-button' disabled={disabled} onClick={onClick}>{lable}</button>
```

Comment:
Instead of label I would use children.
There is a react type 'PropsWithChildren' you can use for that.